### PR TITLE
Fix location of conf.py in readthedocs config

### PR DESCRIPTION
**Pull request by jtsay362**
I had to do this to get the docs to build successfully in readthetodocs.io

### CODE REVIEW

File: {{cookiecutter.project_slug}}/.readthedocs.yml [hunk 1]
```diff
@@ -6,13 +6,13 @@ version: 2
 
 # Build documentation in the docs/ directory with Sphinx
 sphinx:
-  configuration: docs/conf.py
+  configuration: docs/source/conf.py
```
Comment (browniebroke) [inline {{cookiecutter.project_slug}}/.readthedocs.yml:9]: Nice catch 👍

File: {{cookiecutter.project_slug}}/.readthedocs.yml [hunk 2]
```diff
@@ -6,13 +6,13 @@ version: 2
...
 
 # Optionally set the version of Python and requirements required to build your docs
 python:
   version: 3.8
   install:
     - method: pip
-      path: .
+      path: .git@github.com:browniebroke/cookiecutter-pypackage.git
```
Comment (browniebroke) [inline {{cookiecutter.project_slug}}/.readthedocs.yml:16]: Sounds like this was added by mistake?

```suggestion
      path: .
```

**Review by browniebroke — COMMENTED** [review]
Thanks for the fix.

File: {{cookiecutter.project_slug}}/.readthedocs.yml (hunk 2 above)
Comment (browniebroke) [reply {{cookiecutter.project_slug}}/.readthedocs.yml:16]: Fixed in #178

**Comment by browniebroke** [comment]
Thanks for the contribution 🎉

File: {{cookiecutter.project_slug}}/.readthedocs.yml (hunk 2 above)
Comment (jtsay362) [reply {{cookiecutter.project_slug}}/.readthedocs.yml:16]: Oops, sorry!

**Pull request merged**